### DISCUSSION TITLE
I have multiple computers/servers running plotters..  I can only have one node so how does this handle that, or does this run a full node on all instances?

### DISCUSSION THREAD

**Discussion by danktankk** [swar/Swar-Chia-Plot-Manager#122] [Ideas]
I have multiple computers/servers running harvesters/plotters..  I can only have one node so how does this handle that, or does this run a full node on all instances of it?  My question i guess would be how would I make this ONLY harvest and not do anything more than that?

I really like the interface and way it is set up, but if I cant use it on more than one PC, it wont do me much good

A suggestion would be to create a way to monitor in real time all harvesters in the network to provide a display similar to what you currently have.

**Comment by raid5** [comment]
This tool is only for managing plotting, it doesn't interface with the full node, harvester, etc.

**Reply by danktankk** [reply]
it has to connect with something.  forgive my ignorance, but I would like to use this tool everywhere except for the main pc that I use to run the full node.  and the harvester i thought is the thing that makes the plots?

**Reply by tooeffayy** [reply]
It just runs the 'chia plots create' command and keeps track of how many are running. You don't need to touch the harvester, farmer, etc. for that.
